### PR TITLE
Merge 4.41.0 and upgrade schema-utils

### PR DESCRIPTION
**Pull request by sokra**
need update in schema-utils @evilebottnawi 

**What kind of change does this PR introduce?**
merge

**Did you add tests for your changes?**
existing tests

**Does this PR introduce a breaking change?**
no

**What needs to be documented once your changes are merged?**
nothing

### CODE REVIEW

File: lib/optimize/LimitChunkCountPlugin.js [hunk 1]
```diff
@@ -46,53 +69,174 @@ class LimitChunkCountPlugin {
...
+								sizeDiff: aSize + bSize - integratedSize,
+								integratedSize,
+								a,
+								b,
+								aIdx,
+								bIdx,
+								aSize,
+								bSize
+							};
+							combinations.add(c);
```
Comment (vankop) [inline lib/optimize/LimitChunkCountPlugin.js]: @sokra I am just curious why you use here lazy sorting? Looks like inserting in right order will be faster.

Comment (vankop) [reply lib/optimize/LimitChunkCountPlugin.js]: Ah, you deleting unprocessed combinations later

**Comment by evilebottnawi** [comment]
WIP on `not`

**Comment by evilebottnawi** [comment]
Done in https://github.com/webpack/schema-utils/releases/tag/v2.3.0

**Comment by evilebottnawi** [comment]
@adamkeinan please stop spamming unhelpful messages, thanks

**Comment by sokra** [comment]
Thanks

**Comment by adamkeinan** [comment]
> @adamkeinan please stop spamming unhelpful messages, thanks

saw  this message now. im really sorry,dont really know what you talk about cause i hardly comment at all, but if there was repeated spam messages i do apologize.

**Pull request merged**